### PR TITLE
Update and improve the README

### PR DESCRIPTION
**Pull request by officialcjunior**
Update the repository's README.

Changes made:
* Rephrased some sentences.
* Removed the CI status indicator since we haven't added Github CI yet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,34 @@
...
 * Depends on external libraries (yara, ewf)
-* Marginal/specific use (ctf-specific asm/anal vm f.ex)
-* r2 must be self-contained (no optional/external deps)
-* Simplify packagers work, and makes it more flexible
-* Duplicates functionality from r2 (no need to have
-  multiple disassemblers for the same arch in core)
-
-The compiled plugins will be installed at
+* Marginal/specific use (ctf-specific assembly/analalysis vm f.ex)
+* Rizin must be self-contained (no optional/external deps)
```
Comment (XVilka) [inline README.md:9]: Can be removed since it's not really true anymore.

File: README.md [hunk 2]
```diff
@@ -1,39 +1,34 @@
...
+* Duplicates functionality from Rizin (no need to have
+  multiple disassemblers for the same architecture in core)
 
-	/usr/lib/radare2-extras/$(VERSION)
+Depending on the type of the plugin (user or library), the compiled
+plugins will be installed at the location `RZ_LIBR_PLUGINS` or `RZ_USER_PLUGINS`
+is pointing to. See `rizin -H` for more information about these variables.
 
-Some of the plugins/code doesnt compiles, it will be reviewed
+Some of the plugins/code doesn't compile, it will be reviewed
```
Comment (XVilka) [inline README.md]: "don't compile"

**Review by XVilka — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (officialcjunior) [reply README.md:9]: All four of them or just the first two?

Comment (XVilka) [reply README.md:9]: Just the one.

**Pull request merged**